app/src/main/java/com/everest/moviedb/adapters/PageAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package com.everest.moviedb.adapters
+
+import android.content.Context
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentManager
+import androidx.fragment.app.FragmentStatePagerAdapter
+import com.everest.moviedb.R
+import com.everest.moviedb.ui.LatestMovieFragment
+import com.everest.moviedb.ui.PopularMovieFragment
+
+
+class PageAdapter(fragmentManager: FragmentManager, context: Context) :
+    FragmentStatePagerAdapter(fragmentManager) {
+
+    var pageTitles: Array<String> = context.resources.getStringArray(R.array.tab_titles)
```

Comment:
Instead of passing context and then fetching string array, you can just pass string array right?